skype/skype-ps/skype/Set-CsTeamsAppPermissionPolicy.md

```diff
@@ -138,11 +204,12 @@ Accept wildcard characters: False
 ```
 
 ### -GlobalCatalogAppsType
-Do not use.
+Choose to allow or block the installation of third-party apps.
+
 
 ```yaml
 Type: String
-Parameter Sets: (All)
+Parameter Sets: AllowedAppList,BlockedAppList

```

Comment:
The parameters sets in the article are Identity and Instance, are these that you are mentioning here, valid parameter sets?

Reply:
yes, you can choose one of them as GlobalCatalogAppsType

Reply:
One thing is a parameter set name and the other a parameter value